### ISSUE TITLE
Plugin not working on Meta Quest 3

### ISSUE THREAD

**Issue by DomCoccorese** [sccn/labstreaminglayer#124]
Hello
I'm using this plugin inside an Unreal Engine project where I use an outlet component to send a simple message. Although it works in preview, the resulting apk installed in my Oculus Meta Quest 3 does not send any message. I want to specify that the Headset and the computer (where the inlet program is installed) are connected to the same network. Am I missing something? Does anyone faced the same issue with Meta Quest? 

Thanks in advice

**Comment by ChadChen68** [comment]
Are you fixed that problem yet? Cause I got the same problem here, I use unity  and it's everything ok at HMD but after I publish to meta quest pro my stream are missing and object are not showing.

**Comment by DomCoccorese** [comment]
Turned out that the plugin is not compatible with adroid devices. I ended
up by writing a sort of middleware that translates OSC message to LSL on
Windows.

Il giorno lun 25 nov 2024 alle ore 10:18 ChadChen68 <
***@***.***> ha scritto:

**Comment by dmedine** [comment]
Did you try this?
https://github.com/labstreaminglayer/liblsl-Android/tree/675a913b9197998d8525578e74f65e1154b9547a

I have never used it, but it should be the case that you can use LSL on Android using those wrappers.

**Comment by DomCoccorese** [comment]
Yes I have seen that lib, unfortunately I didn't manage to make it work.
Instead I've made a simple application in Unreal using the OSC plugin which
reads messages from the HMD and then translates them to LSL messages.

Il giorno lun 25 nov 2024 alle ore 21:56 David Medine <
***@***.***> ha scritto:
